### DISCUSSION TITLE
Suggestions for the UI

### DISCUSSION THREAD

**Discussion by Akascape** [ntsc-rs/ntsc-rs#82] [Ideas]
Really a great software for VHS simulation, I want to add some suggestion for the UI part:
- Add drag and drop file feature
- Add close/cancel messagebox before exiting the software
- Bug: there are some random flickering in the UI
- show tooltips only at the label part not at slider
- Double click on the label to reset the slider value
- Add minimum width/height for the app when resized
- Loop button for video

**Comment by Degamisu** [comment]
I would recommend moving the bug to the issues tab

I wouldnt recommend removing tooltips, minimum width and height, and double click. Why:

Tooltips are useful. it is more of a description
What if your width and height is needed to go under that minimum?
The sliders dont reset, making them not have a "default"

**Comment by Akascape** [comment]
> I would recommend moving the bug to the issues tab

@Degamisu Ok, I will report this in the issue tab

> I wouldnt recommend removing tooltips, minimum width and height, and double click. Why:

I mean the tooltip should only display at the label/text part, which is on the right side, currently it is also showing over the slider when hovered. But no issue if it's intentional.

> What if your width and height is needed to go under that minimum?

No, I mean to say that the window should have a minimum size from where it should not get resized further. 

![image](https://github.com/valadaptive/ntsc-rs/assets/89206401/d29885bd-ab8c-4a44-b16c-c34f52269e55)

>The sliders dont reset, making them not have a "default"

But there is a reset all button in the bottom, I guess it is just loads a json file with some default values, but maybe you can try getting one value from there and reset the particular slider. Just a suggestion.

I hope you understand :)

**Reply by Degamisu** [reply]
oh ok! i thought you meant the actual video. Understood.

**Comment by smfthe1199** [comment]
> Really a great software for VHS simulation, I want to add some suggestion for the UI part:
> 
> * Add drag and drop file feature
> * Add close/cancel messagebox before exiting the software
> * Bug: there are some random flickering in the UI
> * show tooltips only at the label part not at slider
> * Double click on the label to reset the slider value
> * Add minimum width/height for the app when resized
> * Loop button for video

I do agree tho!

**Reply by smfthe1199** [reply]
Well, the drag-and-drop suggestion anyway!

**Reply by valadaptive** [reply]
Please refrain from commenting things like "I agree" across a wide range of discussion threads. It results in a bunch of notifications for me as well as everyone who's commented in the thread. If you want to express your agreement and support, an upvote or perhaps a thumbs-up reaction is a good alternative.

**Reply by smfthe1199** [reply]
Oh ok! Sry for disturbance!
